druid/examples/disabled.rs

```diff
@@ -0,0 +1,93 @@
+use druid::widget::{Checkbox, CrossAxisAlignment, Flex, Label, Slider, Stepper, Switch, TextBox};
```

Comment:
I think we still include the license with examples?

Reply:
Ok :)